package.json

```diff
@@ -8,11 +8,15 @@
     "start": "next start"
   },
   "dependencies": {
+    "@types/d3": "^5.16.3",
+    "@types/react-icons": "^3.0.0",
     "axios": "^0.20.0",
+    "d3": "^6.2.0",
     "dotenv": "^8.2.0",
     "next": "9.5.3",
     "react": "16.13.1",
     "react-dom": "16.13.1",
+    "react-icons": "^3.11.0",
```

Comment:
we will use icons from material-ui, so we shouldn't import whole library just to use 2 images.